main.py

```diff
@@ -177,7 +180,10 @@ def var(self, name):
 
     def store_var(self, name):
         name = self.mangle(name)
-        sym = self.table.lookup(name)
+        try:
+            sym = self.table.lookup(name)
```

Comment:
These were needed because I'm inserting assignments to names that weren't in the original source, which means they aren't in the symbol table.